### PR TITLE
labhub: Include link to gitter message in iss body

### PR DESCRIPTION
**Pull request by meetmangukiya**
Closes https://github.com/coala/corobo/issues/147

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: plugins/labhub.py [hunk 1]
```diff
@@ -136,9 +136,18 @@ def create_issut_cmd(self, msg, match):
         repo_name = match.group(1)
         iss_title = match.group(2)
         iss_description = match.group(3) if match.group(3) is not None else ''
+
+        try:  # This is gitter backend specific
+            link = '\nOpened via [gitter](https://gitter.im/{}?at={})'.format(
```
Comment (meetmangukiya) [inline plugins/labhub.py]: Add username as well, of the user that created the issue, so he remains in the loop

**Comment by meetmangukiya** [comment]
ack 1ed9bc135c7a0fbb0eaca4ca891581ed84e152bc

**Comment by meetmangukiya** [comment]
ack 9adf59326d74255f05dac5132208fc8fe43d523f

**Comment by meetmangukiya** [comment]
f39f5cb6a1c04d919e3f200831347e73db48bc81 reduces the length of help message to quite some extent

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/154/commits/8d9d7a974b165ce1138f5fa65191a869ff32b3df

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**